test/packages/parallel/httpcheck/_dev/test/system/test-local-config.yml

```
@@ -0,0 +1,7 @@
+skip:
+  reason: Not supported system tests with input type otelcol.
+  link: https://github.com/elastic/elastic-package/issues/2835
+service: web
+vars:
```

Nit. I found out that setting a period here makes tests faster :sweat_smile: It defaults to 60s.
```suggestion
vars:
  period: 1s
```